Pages/Product page/product.css

```diff
@@ -205,31 +209,116 @@
 	color: var(--primary-color);
 }
 
+.filter-navigation-small-screen {
+    display: none;
+}
+
 /*Media-Queries*/
 
-/* @media screen and (max-width : 1140px) {
-    :root{
-        font-size: 14px;
+@media screen and (max-width : 1160px) {
+
+    .navbar {
+        padding-top: 1rem;
+        padding-bottom: 1rem;
+    }
+
+    .nav-icons {
+        flex-grow: 1;
+        justify-content: flex-end;
+    }
+
+    .searchbar {
+        display: none;
+    }
+
+    .searchbar-small-screen {
+        display: flex;
+        z-index: 1;
+        position: fixed;
+        top: 4.4rem;
+        width: 100%;
+        border: none;
+    }
+
+    .footer {
+        z-index: 1;
+        min-width: 35.7rem;
+        padding-top: 1rem;
+        padding-bottom: 1rem;
     }
 }
 
 @media screen and (max-width : 1000px) {
     :root{
-        font-size: 12px;
+        font-size: 14px;
     }
 }
 
-@media screen and (max-width : 850px) {
-    :root{
-        font-size: 10px;
+@media screen and (max-width : 880px) {
```

Comment:
This can be merged in 850px media queries.

Reply:
Yes will update this. Thanks for reviewing.